### PR TITLE
cherry pick core_ext

### PR DESCRIPTION
**Pull request by chriscummings**
Recurrence was throwing an exception upon loading.

screen shot 2015-10-29 at 11 34 10 pm

I found the following discussion https://github.com/rails/rails/issues/14664, and applied their fix. Tests run green and the issue is resolved for me.

### CODE REVIEW

**Comment by chriscummings** [comment]
Also, this library is great. This is going to save me a lot of headache.

File: lib/recurrence/namespace.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require 'active_support'
```
Comment (fnando) [inline lib/recurrence/namespace.rb]: Can you change it to double quotes?

**Comment by chriscummings** [comment]
Good god, you're right. I didn't even notice.

**Comment by fnando** [comment]
Can you also squash you two commits into one? :)

**Comment by chriscummings** [comment]
No.

**Pull request closed without merging**